shared/js/background/declarative-net-request.js

```diff
@@ -0,0 +1,209 @@
+/** @module declarativeNetRequest */
```

Comment:
What does this do out of interest? I think we have the right tsconfig flags not to need this but happy to be wrong :)

Reply:
I'm not sure honestly, I always added that in the past but if you don't think it's necessary then I'm happy to remove. Done.